modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -8,20 +8,20 @@ process ENSEMBLVEP_VEP {
         'quay.io/biocontainers/ensembl-vep:108.2--pl5321h4a94de4_0' }"
 
     input:
-    tuple val(meta), path(vcf)
+    tuple val(meta), path(vcf), path(custom_extra_files)
```

Comment:
what would these custom extra files be?

Reply:
We're trying to implement SV annotation in a pipeline. This will be done using AnnotSV, VEP and VCFanno. AnnotSV only emits the fields that the tool itself uses and outputs (and removes other fields from the VCF). So we came up with the idea to use `-custom` to add the annotation fields from the AnnotSV VCF to the annotated VCF from VEP to retain all original fields along with the annotation fields. (We've changed on our mind in the meantime and will add the annotation fields from AnnotSV during the VCFanno step, but I think it can still be a useful addition)

Reply:
In our case, yes! All non-sample specific files should be entered through `vep_extra_files` in any case IMO.